src/widget/widget.js

```diff
@@ -20,22 +20,35 @@ export class Widget extends React.Component {
         "https://www.generatormix.com/images/thumbs/random-car-model-generator.jpg"
       ],
       loading: false,
+        inputData: {clientIDs: "1402110922112412", tradeInCar: "2018 Ford Focus"},
+        success: false
     };
   }
+  sendRequest = () => {
+        const requestOptions = {
+            method: 'PUT',
+            headers: {'Content-type': 'application/json'},
+            body: JSON.stringify(this.state.inputData)
+        };
+        fetch('https://cb.caravantage.tech/generateCars',requestOptions)
+            // Handle success
+            .then(result => {
+                this.setState({success: result});

```

Comment:
a better way of handling success is response.ok

You can check how i did this in the TLI-client - the idea is the response has some properties you can use (ok is basically your success state)

Reply:
Fixed the issue